AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/handler/ItemSelectHandler.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (C) 2010-2014 eBusiness Information, Excilys Group
- *
+ * <p/>
```

Comment:
Strange, I didn't add them manually.
I pushed my commit late yesterday night so that you can review it but I didn't take time to review the diff.
Feel free to modify my commit. If you don't want to, I will change when I have time.